chat-client-ui-types/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "@aws/chat-client-ui-types",
+    "version": "0.0.1",
+    "description": "Type definitions for Chat UIs in Language Servers and Runtimes for AWS",
+    "main": "index.js",
+    "scripts": {
+        "clean": "rm -rf out/",
+        "compile": "tsc --build",
+        "prepub:copyFiles": "cp ../.npmignore CHANGELOG.md ../LICENSE ../NOTICE README.md ../SECURITY.md package.json out/",
+        "prepub": "npm run clean && npm run compile && npm run prepub:copyFiles",
+        "pub": "cd out && npm publish"
```

Comment:
nit: for simple npm package (not monorepo), it's enough and more strainghtforward to publish from root of package, instead of `/out` directory. Not critical, but would be nice to simplify.

We had to publish `runtimes` from `out` for flatten import paths for multiple subfolders. Downside of this is that `npm link` has to be performed from `/out` dir as well during development, which is a bit more confusing.

Reply:
I see, that makes sense. I will create new PR to update this

Reply:
https://github.com/aws/language-server-runtimes/pull/114